webapp/channels/src/actions/post_actions.ts

```diff
@@ -127,6 +136,22 @@ export function createPost(post: Post, files: FileInfo[], afterSubmit?: (respons
     };
 }
 
+export function createSchedulePostFromDraft(scheduledPost: ScheduledPost): ActionFuncAsync<PostActions.CreatePostReturnType, GlobalState> {
```

Comment:
Its doing so because I need process the error occurring in creating the scheduled posts and for type correctness. If I return something else, I need to update the type in a lot of places, basically in all the functions the entire chain of post button till the API call and the same in post/edit comment. Doing so wasn't providing any benefit.